src/main/java/net/fabricmc/loom/util/SourceRemapper.java

```diff
@@ -61,13 +61,7 @@ public SourceRemapper(Project project, boolean toNamed) {
 		this.toNamed = toNamed;
 	}
 
-	public static void remapSources(Project project, File input, File output, boolean named, boolean reproducibleFileOrder, boolean preserveFileTimestamps) {
-		SourceRemapper sourceRemapper = new SourceRemapper(project, named);
-		sourceRemapper.scheduleRemapSources(input, output, reproducibleFileOrder, preserveFileTimestamps);
-		sourceRemapper.remapAll();
-	}
-
-	public void scheduleRemapSources(File source, File destination, boolean reproducibleFileOrder, boolean preserveFileTimestamps) {
+	public void scheduleRemapSources(File source, File destination, boolean reproducibleFileOrder, boolean preserveFileTimestamps, Runnable compilationCallback) {
```

Comment:
Is the runnable ever used? There don't seem to be any changes doing that

Reply:
Good spot, forgot to invoke it 👍

Reply:
and its named wrong